src/test/java/org/springframework/social/bitbucket/api/impl/GroupPrivilegesTemplateTest.java

```diff
@@ -87,7 +87,7 @@ public void testGetPrivilegedGroupsForRepository() throws Exception {
         assertNotNull(firstPrivilegeGroup.getGroup().getMembers());
         assertTrue(firstPrivilegeGroup.getGroup().getMembers().isEmpty());
         assertEquals("Contractors", firstPrivilegeGroup.getGroup().getName());
-        assertTrue(firstPrivilegeGroup.getRepository().getOwner().isTeam());
+        assertTrue(firstPrivilegeGroup.getRepository().getOwner().getIsTeam());
```

Comment:
This `getIsTeam()` is an ugly name. We should probably get rid of Lombok to enable a better name.